src/ts/translations/de.ts

```diff
@@ -0,0 +1,41 @@
+import {Translation} from './translation-types';
+
+let lang: Translation = {
```

Comment:
I like the general idea of the structure of this `Translation` definition, I'd flatten the hierarchy a bit though

Reply:
Also you can directly do a `export default <Translation>{` here